### ISSUE TITLE
Enable settings only for certain keyboard

### ISSUE THREAD

**Issue by Grazfather** [tekezo/Karabiner#774]
I have a keyboard that I use with customizable firmware, but I also often use the laptop keyboard. I can program the keyboard with Colemak, but unfortunately I cannot the keyboard firmware. This makes it so that I either have to do something like #454, or keep the normal firmware.

I suppose this is a feature request to somehow allow certain settings to only apply to certain keyboards (in my case "Change QWERTY to Colemak" should only apply to my laptop's keyboard).

**Comment by dunkarooftop** [comment]
Filter: Specify device

https://pqrs.org/osx/karabiner/xml.html.en#devicedef

**Comment by Grazfather** [comment]
Yes, but it seems this is something many would want: You shouldn't have to customize your xml for every option you only want to apply to one keyboard.

**Comment by dunkarooftop** [comment]
maybe use profile ?

**Comment by Grazfather** [comment]
Yes the idea is to not need to switch profiles, and instead allow features be set based on vendor and/or device id. I can work around it: This is a feature request.
